client/bundles/Main/components/sessions/AddSession.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import ReactOnRails from "react-on-rails";
+import AddSessionForm from "./AddSessionForm";
+
+const axios = require("axios");
+
+export class AddSession extends Component {
+	
+	constructor(props) {
+		super(props);
+		this.state = {
+			title: "",
+			start_date: "",
+			price_point: 1,
```

Comment:
why hard code '1'?

Reply:
We need to refactor our price table to change this, we have created a chore for this in our PT.